deployment/.env.template

```diff
@@ -15,17 +15,21 @@ PFS_HOST=0.0.0.0
 PFS_KEYSTORE_FILE=/keystore/<FILENAME>
 PFS_PASSWORD=<PASSWORD>
 PFS_LOG_LEVEL=DEBUG
+# PFS_ACCEPT_DISCLAIMER=TRUE
```

Comment:
Should we have the disclaimer somewhere so that we add a link here?

Reply:
@Dominik1999

Reply:
Yes, can we simply link to the constants.py on gitHub?

Reply:
atm we do not need to link that. We need to come up with a holistic view on which disclaimers to put where